samza-sql/src/test/java/org/apache/samza/sql/util/MyTestObjUdf.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.samza.sql.util;
+
+import org.apache.samza.config.Config;
+import org.apache.samza.sql.schema.SamzaSqlFieldType;
+import org.apache.samza.sql.udfs.SamzaSqlUdf;
+import org.apache.samza.sql.udfs.SamzaSqlUdfMethod;
+import org.apache.samza.sql.udfs.ScalarUdf;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Test UDF used by unit and integration tests.
+ */
+@SamzaSqlUdf(name = "MyTestObj", description = "Test UDF.")
+public class MyTestObjUdf implements ScalarUdf {
+
+  private static final Logger LOG = LoggerFactory.getLogger(MyTestUdf.class);
```

Comment:
Why we use `MyTestUdf.class` here? What is the relationship between "MyTestObjUdf" class and `MyTestUdf` class? I saw the method `execute(Object value)` is deleted from `MyTestObj` class.

Reply:
This was a mistake, I will fix this. there is no relationship between MyTestUdf and MyTestObj. the deleted method in MyTestUDF was unused.